edn/src/edn.rustpeg

```diff
@@ -98,17 +98,17 @@ pub keyword -> Value =
     }
 
 pub list -> Value =
-    "(" v:(value)* ")" {
+    "(" __ v:(value)* __ ")" {
```

Comment:
Oh, interesting, I thought I'd covered this with https://github.com/mozilla/mentat/blob/90c1c59881b1bbef1394961e86add1d95c363617/edn/src/edn.rustpeg#L128 but if there are no values, then you need the whitespace.  Good catch!